packages/augur-ui/src/modules/reporting/components/markets-in-dispute.tsx

```diff
@@ -125,6 +133,10 @@ export default class MarketsInDispute extends Component<
     }
   }
 
+  componentWillUnmount() {
+    console.log("component has unmounted");
+  }
```

Comment:
Should this be removed?

Reply:
I need it for troubleshooting. every once in a while markets-in-dispute blows up.

Reply:
I think something is unmouting it while a load markets call is returning values. still thinking about it.